sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi_simple.c

```diff
@@ -0,0 +1,387 @@
+#endif
+
+// these parameters are used in the filtering of the angular acceleration
+// define them in the airframe file if different values are required
+#ifndef STABILIZATION_INDI_FILT_OMEGA
+#define STABILIZATION_INDI_FILT_OMEGA 50.0
+#endif
+
+#ifndef STABILIZATION_INDI_FILT_ZETA
+#define STABILIZATION_INDI_FILT_ZETA 0.55
+#endif
+
+// the yaw sometimes requires more filtering
+#ifndef STABILIZATION_INDI_FILT_OMEGA_R
+#define STABILIZATION_INDI_FILT_OMEGA_R STABILIZATION_INDI_FILT_OMEGA
+#endif
+
+#ifndef STABILIZATION_INDI_FILT_ZETA_R
+#define STABILIZATION_INDI_FILT_ZETA_R STABILIZATION_INDI_FILT_ZETA
+#endif
+
+#ifndef STABILIZATION_INDI_MAX_RATE
+#define STABILIZATION_INDI_MAX_RATE 6.0
+#endif
+
+#if STABILIZATION_INDI_USE_ADAPTIVE
+#warning "Use caution with adaptive indi. See the wiki for more info"
+#endif
+
+#ifndef STABILIZATION_INDI_MAX_R
+#define STABILIZATION_INDI_MAX_R STABILIZATION_ATTITUDE_SP_MAX_R
+#endif
+
+struct Int32Eulers stab_att_sp_euler;
+struct Int32Quat   stab_att_sp_quat;
+
+static int32_t stabilization_att_indi_cmd[COMMANDS_NB];
+static inline void stabilization_indi_calc_cmd(int32_t indi_commands[], struct Int32Quat *att_err, bool rate_control);
+static void stabilization_indi_second_order_filter_init(struct IndiFilter *filter, float omega, float zeta, float omega_r);
+static void stabilization_indi_second_order_filter(struct IndiFilter *filter, struct FloatRates *input);
+static inline void lms_estimation(void);
+
+#define INDI_EST_SCALE 0.001 //The G values are scaled to avoid numerical problems during the estimation
+struct IndiVariables indi = {
+  .max_rate = STABILIZATION_INDI_MAX_RATE,
+  .attitude_max_yaw_rate = STABILIZATION_INDI_MAX_R,
+
+  .g1 = {STABILIZATION_INDI_G1_P, STABILIZATION_INDI_G1_Q, STABILIZATION_INDI_G1_R},
+  .g2 = STABILIZATION_INDI_G2_R,
+  .reference_acceleration = {
+    STABILIZATION_INDI_REF_ERR_P,
+    STABILIZATION_INDI_REF_ERR_Q,
+    STABILIZATION_INDI_REF_ERR_R,
+    STABILIZATION_INDI_REF_RATE_P,
+    STABILIZATION_INDI_REF_RATE_Q,
+    STABILIZATION_INDI_REF_RATE_R},
+
+  /* Estimation parameters for adaptive INDI */
+  .est = {
+    .g1 = {
+      STABILIZATION_INDI_G1_P / INDI_EST_SCALE,
+      STABILIZATION_INDI_G1_Q / INDI_EST_SCALE,
+      STABILIZATION_INDI_G1_R / INDI_EST_SCALE},
+    .g2 = STABILIZATION_INDI_G2_R / INDI_EST_SCALE,
+    .mu = STABILIZATION_INDI_ADAPTIVE_MU,
+  },
+
+#if STABILIZATION_INDI_USE_ADAPTIVE
+  .adaptive = TRUE,
+#else
+  .adaptive = FALSE,
+#endif
+};
+
+#if PERIODIC_TELEMETRY
+#include "subsystems/datalink/telemetry.h"
+
+static void send_att_indi(struct transport_tx *trans, struct link_device *dev)
+{
+  //The estimated G values are scaled, so scale them back before sending
+  struct FloatRates g1_disp;
+  RATES_SMUL(g1_disp, indi.est.g1, INDI_EST_SCALE);
+  float g2_disp = indi.est.g2 * INDI_EST_SCALE;
+
+  pprz_msg_send_STAB_ATTITUDE_INDI(trans, dev, AC_ID,
```

Comment:
Make sure the message is consistent with the one defined in new INDI, or use a different message